torch/functional.py

```diff
@@ -403,3 +404,36 @@ def argmin(input, dim=None, keepdim=False):
     if dim is None:
         return torch._argmin(input.contiguous().view(-1), dim=0, keepdim=False)
     return torch._argmin(input, dim, keepdim)
+
+
+def argsort(input, dim=None, descending=False):
```

Comment:
I think this should be just as simple as:
```
def argsort(input, *args):
    return torch.sort(input, *args)[1]
```

Reply:
argsort does not accept the output parameter that torch.sort does, so this doesn't work.

Reply:
Oh yes, you're right.